src/completers/git.zsh

```diff
@@ -152,7 +169,17 @@ _fzf_complete_git() {
                 ;;
 
             *)
-                _fzf_complete_git-ls-files '' '--multi' $@
+                if [[ -n ${${(Q)${(z)arguments}}[(r)--source]} ]] || [[ -n ${${(Q)${(z)arguments}}[(r)-[^-]#s]} ]]; then
```

Comment:
Question: Does it accept `-sX` as well?

Reply:
No, don't accept. You can make sure by running below command.

```sh
echo ${$(echo -Xs)[(r)-[^-]#s]}
echo ${$(echo -sX)[(r)-[^-]#s]}
```

Reply:
Then here needs to be fixed to accept it.